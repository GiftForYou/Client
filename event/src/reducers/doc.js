export const initialState = {
  doc_user:{},
  doc_detail:[],
  doc_recomendation:[]
};

export default (state = initialState, action) => {
  switch (action.type) {
    case 'SET_DOC_USER':
      return Object.assign({}, state, {doc_user: action.value});
    case 'SET_DOC_DETAIL':
      return Object.assign({}, state, {doc_detail: action.value});
    case 'SET_RECOMEND':
      return Object.assign({}, state, {doc_recomendation: action.value});
    default:
      return state;
  }
};
