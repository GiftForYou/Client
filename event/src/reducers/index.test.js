import React from 'react';
import {combineReducers} from 'redux';
import doc from './doc';

test('[Event Test] overAll index', () => {
  expect(combineReducers({doc})).toMatchSnapshot();
});
