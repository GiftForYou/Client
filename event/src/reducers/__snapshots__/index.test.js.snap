// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Event Test] overAll index 1`] = `[Function]`;
